dist/purchaserInfo.d.ts

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
The naming of this file made me thing... should we use Lower Camel Case for the files? I don't think there's a convention and I don't really care about the naming, just wanted to share my thoughts in case you had an opinion or had look into it.

Reply:
I was actually unsure about that too, I'll check conventions

Reply:
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#filename